frontend/index.html

```diff
@@ -4,6 +4,9 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
Could you move this 3 lines of fonts import in index.css file. I was import Jost and Inter fonts right in index.css.